UserGuideComprehensive.adoc

```diff
@@ -1328,22 +1328,22 @@ process and thus reduce the network load of transmitting the given data to the c
 
 A constraint expression is appended to the target URL following a question mark, as in the following examples:
 
-.../nc/sst.mnmean.nc.gz?sst[3][4][5] http://test.opendap.org/dap/data/nc/sst.mnmean.nc.gz.asc?sst%5B3%5D%5B4%5D%5B5%5D[Click here for sample output.]
+.../nc/sst.mnmean.nc.gz?sst[3][4][5] http://test.opendap.org/dap/data/nc/sst.mnmean.nc.gz.ascii?sst[3][4][5][Click here for sample output.]
```

Comment:
Does the documentation describe why some of the characters in the URL need to be escaped? If not, this might be a good place to do that.

It might also be good to show all of the URL unescaped but then actually send the escaped content to the server. Maybe that's the case already and I just cannot see it because I'm looking at a diff of English text...

Reply:
> Does the documentation describe why some of the characters in the URL need to be escaped? If not, this might be a good place to do that.

I don't think the documentation describes anything about escaping characters in URLs. But my understanding is that the escaping of characters is not something `users` have to do, right? But I certainly write a note describing the escaping of characters

> It might also be good to show all of the URL unescaped but then actually send the escaped content to the server. Maybe that's the case already and I just cannot see it because I'm looking at a diff of English text...

Yeah sure. BTW I found that to visualize ascii data on the browser the unscaped URL works: [http://test.opendap.org/dap/data/nc/sst.mnmean.nc.gz.ascii?sst[3][4][5]](http://test.opendap.org/dap/data/nc/sst.mnmean.nc.gz.ascii?sst[3][4][5]).

The way it shows on text is unescaped. The hyperlink used was an escaped URL but I changed it to unescaped because I found that it actually works if unescaped too. I will return it to escaped